doctr/__main__.py

```diff
@@ -89,8 +89,7 @@ def get_parser():
         public repositories for the user. This option is not recommended
         unless you are using a separate GitHub user for deploying.""")
     configure_parser.add_argument("--no-upload-key", action="store_false", default=True,
-        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub. If you select this
-        option, you will not be prompted for your GitHub credentials. """)
+        dest="upload_key", help="""Don't automatically upload the deploy key to GitHub.""")
```

Comment:
Maybe we can ask if the repo is private if it doesn't seem to exist, and then prompt?